realm/realm-library/src/main/java/io/realm/log/Logger.java

```diff
@@ -16,75 +16,23 @@
 
 package io.realm.log;
 
+import io.realm.internal.KeepMember;
+
 /**
  * Interface for custom loggers that can be registered at {@link RealmLog#add(Logger)}.
  * The different log levels are described in {@link LogLevel}.
  */
 public interface Logger {
```

Comment:
This interface is in the public API, we cannot just change it in a minor release without warning :(